pages/[province].js

```diff
@@ -157,8 +157,8 @@ function ProvincePage(props) {
           )}
           {searchValue.length > 1 && hospitals && hospitals.length < 1 && (
             <Text textAlign="center" w="100%" p="24" color="gray.600">
-              Pencarian rumah sakit dengan keyword "{searchValue}" tidak
-              ditemukan.
+              Pencarian rumah sakit dengan keyword &quot;{searchValue}&quot;
+              tidak ditemukan.
```

Comment:
Hmmm. Please replace this with
```js
{`Pencarian rumah sakit dengan keyword "{searchValue}" tidak ditemukan.`}
```

Reply:
got it

Reply:
Hi @agallio, just to make sure, u means like `{Pencarian rumah sakit dengan keyword "${searchValue}" tidak ditemukan.}` ?.

Because if wrote `{Pencarian rumah sakit dengan keyword "{searchValue}" tidak ditemukan.}` then will rendering `Pencarian rumah sakit dengan keyword "{searchValue}" tidak ditemukan.`

See:
![image](https://user-images.githubusercontent.com/28673206/126267479-137bbea6-62cf-4abd-9106-04c45053b735.png)

Reply:
Yes, you're right! Sorry, my bad.

Reply:
got it. See next commit.